package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-reflex",
-  "version": "4.2.4",
+  "version": "4.2.6",
```

Comment:
Bumping to `4.2.6` as the latest version on NPM is `4.25`:

<img width="829" alt="Screenshot 2024-03-10 at 8 08 14 PM" src="https://github.com/leefsmp/Re-Flex/assets/43310906/5cd032f7-1139-4d2f-9a90-e2b94c8b9bc8">